CRM/Sepa/BAO/SEPATransactionGroup.php

```diff
@@ -103,8 +103,7 @@ function generateXML ($id = null) {
       LEFT JOIN civicrm_address a ON c.contact_id = a.contact_id AND a.is_primary = 1
       WHERE g.txgroup_id = %1
         AND c.contribution_status_id != 3
-        AND mandate.is_enabled = true
-      GROUP BY c.id"; //and not cancelled
+        AND mandate.is_enabled = true"; //and not cancelled
```

Comment:
just a remark: ``is_enabled`` has nothing to do with the mandate not being cancelled or not. This is (sometimes) used to keep mandates disabled until they are approved.